src/game/server/gamemodes/base_pvp/base_pvp.cpp

```diff
@@ -1863,6 +1863,11 @@ bool CGameControllerPvp::OnSkinChange7(protocol7::CNetMsg_Cl_SkinChange *pMsg, i
 
 bool CGameControllerPvp::OnTeamChatCmd(IConsole::IResult *pResult)
 {
+	if(!g_Config.m_SvTeam)
+	{
+		return true;
```

Comment:
Does it still print the teams are disabled message? I think this should be return false instead.